src/styles/card.scss

```
@@ -0,0 +1,103 @@
+.card {
+  background-color: $white-accent;
+  font-family: "roboto", sans-serif;
+  color: $main-text-color;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border: solid 1px $border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: 300ms ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+
+  &__description {
+    margin-bottom: 16px;
+
+    .card__title {
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 500;
+      margin: 0 0 4px;
+    }
+
+    .card__code {
+      font-size: 10px;
+      line-height: 14px;
+      font-weight: 400;
+      color: $gray-color;
+      margin: 0;
+    }
+  }
```

do not nest like so, all these blocks can be nested inside `card` block like so
```suggestion
  &__description {
   ...
   }

    &__title {
      ...
    }

    &__code {
      ...
    }
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**

---

Thank you. I was just thinking about how to do it right, because it didn't look good.Your option is better